presto-main/src/test/java/com/facebook/presto/operator/TestPagesIndex.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.SequencePageBuilder;
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.type.Type;
+import com.google.common.collect.ImmutableList;
+import io.airlift.units.DataSize;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+import static com.facebook.presto.spi.type.VarcharType.VARCHAR;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class TestPagesIndex
+{
+    @Test
+    public void testInitialEstimatedSize()
+    {
+        DataSize initialEstimatedSize = newPagesIndex(ImmutableList.of(BIGINT, VARCHAR), 30).getEstimatedSize();
+        assertTrue(initialEstimatedSize.toBytes() > 0, "No initial estimated size");
+    }
+
+    @Test
+    public void testEstimatedSize()
+    {
+        List<Type> types = ImmutableList.of(BIGINT, VARCHAR);
+
+        PagesIndex pagesIndex = newPagesIndex(types, 30);
+        long initialEstimatedSize = pagesIndex.getEstimatedSize().toBytes();
```

Comment:
You can put the `assertTrue` in the above `testInitialEstimatedSize()` here and remove that method.